### PR TITLE
Require tarantool/queue 0.10.

### PR DESCRIPTION
**Pull request by luscinium**
Could you please bump the required queue-version to 0.10?

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
     ],
     "require": {
         "php": "^7.1|^8",
-        "tarantool/queue": "^0.9"
+        "tarantool/queue": "^0.10"
```
Comment (rybakit) [inline composer.json:14]: Hi @luscinium, thanks for the PR. Unfortunately, it is not quite complete. In `tarantool/queue` v0.10, support for PHP 7.1 has been removed, so the PHP version requirements and the `.github/workflows/qa.yml` file must be updated accordingly.  Is this something you're willing to work on?

Comment (luscinium) [reply composer.json:14]: Sure. I've updated the QA-workflow to remove 7.1 and add 8.1. All tests succeeded.

Comment (rybakit) [reply composer.json:14]: Great! There is one minor thing left, the PHP requirement in composer.json needs to be updated to `"php": "^7.2.5|^8"`.

**Comment by rybakit** [comment]
@luscinium thank you!

**Pull request merged**